controllers/grid/settings/sections/SectionGridHandler.inc.php

```diff
@@ -241,15 +241,16 @@ function deleteSection($args, $request) {
 			return new JSONMessage(false, __('manager.setup.errorDeletingItem'));
 		}
 
+		// Validate if it can be deleted
 		AppLocale::requireComponents(LOCALE_COMPONENT_PKP_MANAGER);
 		$submissionDao = DAORegistry::getDAO('SubmissionDAO'); /* @var $submissionDao SubmissionDAO */
-		$checkSubmissions = $submissionDao->retrieve('SELECT p.publication_id FROM publications p JOIN submissions s ON (s.submission_id = p.submission_id) WHERE p.section_id = ? AND s.context_id = ?', array((int) $request->getUserVar('sectionId'), (int) $journal->getId()));
+		$checkSubmissions = $submissionDao->retrieve('SELECT p.publication_id FROM publications p JOIN submissions s ON (s.submission_id = p.submission_id) WHERE p.section_id = ? AND s.context_id = ?', [(int) $request->getUserVar('sectionId'), (int) $journal->getId()]);
```

Comment:
The grid handlers shouldn't be executing SQL directly; this should be moved into the DAO.